addons/api/addon/generated/models/managed-groups.js

```diff
@@ -0,0 +1,55 @@
+import BaseModel from '../../models/base';
+import { attr } from '@ember-data/model';
+
+/**
+ * A managed group is a resource that represents a collection of accounts.
+ */
+export default class GeneratedManagedGroupModel extends BaseModel {
+  // =attributes
+
+  @attr('string', {
+    description: 'Optional name for identification purposes.',
+  })
+  name;
+
+  @attr('string', {
+    description: 'Optional user-set description for identification purposes.',
+  })
+  description;
+
+  @attr('date', {
+    description: 'The time this resource was created',
+    readOnly: true,
+  })
+  created_time;
+
+  @attr('date', {
+    description: 'The time this resource was last updated.',
+    readOnly: true,
+  })
+  updated_time;
+
+  @attr('number', {
+    description: 'Current version number of this resource.',
+  })
+  version;
+
+  @attr('string', {
+    description: 'The type of this ManagedGroup.',
+    defaultValue: () => 'OIDC',
```

Comment:
Should this be defined at mirage level instead of here? Also, api responds with a lowercase `oidc`.

Reply:
You are right. I dunno how I end up putting that in there lol. I already update it. But not sure how to add that at mirage level, I will [check on the other PR](https://github.com/hashicorp/boundary-ui/pull/829) and change it there,